src/app/rundown-view/components/follow-playhead-timeline/follow-playhead-timeline.component.ts

```diff
@@ -77,6 +86,19 @@ export class FollowPlayheadTimelineComponent implements OnChanges {
     }
   }
 
+  private checkForSpanningElements(parts: readonly Part[]): Part[] {
+    return parts.reduce((result: Part[], currentPart: Part, index: number) => {
```

Comment:
What's the reason for using `reduce` and not `map` here?

Reply:
I will include a change to the function using `map` instead :) See if you like that better